v3.0/reference/calicoctl/commands/get.md

```diff
@@ -48,6 +48,9 @@ Options:
                                Uses the default namespace if not specified.
   -a --all-namespaces          If present, list the requested object(s) across 
                                all namespaces.
+  --export                     If present, returns the requested object(s) stripped of
```

Comment:
oh just realized, we don't need to make these changes in `v3.0` docs. It's not in a release yet (the diff from master will get copied when we cut a new release)

Reply:
Not sure about that @gunjan5 . I think that the entire master directory gets copied when we make the initial release candidate. After that, we're on our own. I don't think that there is any diff functionality such as what you describe. But perhaps @bcreane can comment.